packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -282,6 +282,60 @@ describe("Sequence.Revertibles with Local Edits", () => {
 		revertSharedStringRevertibles(sharedString, revertibles.splice(0));
 		assertSequenceIntervals(sharedString, collection, [{ start: 1, end: 3 }]);
 	});
+	it("performs two acked interval removes and reverts to ensure interval returns to correct position", () => {
+		const uRevertibles: SharedStringRevertible[] = [];
+		const rRevertibles: SharedStringRevertible[] = [];
+
+		sharedString.insertText(0, "123456789");
+		collection.add({ start: 2, end: 2 });
+
+		sharedString.on("sequenceDelta", (op) => {
+			if (op.isLocal) {
+				appendSharedStringDeltaToRevertibles(sharedString, op, uRevertibles);
+			}
+		});
+		collection.on("changeInterval", (interval, previousInterval, local, op) => {
+			appendChangeIntervalToRevertibles(
+				sharedString,
+				interval,
+				previousInterval,
+				uRevertibles,
+			);
+		});
+		// remove "34"
+		// gets pushed to urevertibles
+		sharedString.removeRange(2, 4);
+		containerRuntimeFactory.processAllMessages();
+		assert.equal(sharedString.getText(), "1256789");
+		assertSequenceIntervals(sharedString, collection, [{ start: 2, end: 2 }]);
+
+		// revert undo[0]
+		// gets pushed to urevertibles - need to put it on rrevertibles and remove from urevertibles (line 75)
+		revertSharedStringRevertibles(sharedString, uRevertibles.splice(0));
+		containerRuntimeFactory.processAllMessages();
+		rRevertibles.push(...uRevertibles.splice(0));
```

Comment:
I'm not too concerned about this as test code, but probably worth noting that this is not the right way to do this in case any real users references this as how to do undo-redo. There should be tracking of whether or not the actions is an undo and putting it on the right stack. A comment is fine.